webhook_verifier_test.go

```diff
@@ -149,6 +171,9 @@ func TestVerifier_Middleware(t *testing.T) {
 			req := httptest.NewRequest(http.MethodPost, "/", strings.NewReader(c.payload))
 			req.Header.Set("Paddle-Signature", c.signature)
 
+			t.Logf("🧪 Middleware: %s", c.name)
+			t.Logf("📦 Payload size: %d", len(c.payload))
```

Comment:
These look like debug logs that aren't required within the tests, can we remove them and similar in new tests 🙏